ci/post-process.sh

```diff
@@ -7,10 +7,10 @@ set -euo pipefail
 
 CURRENT_DIR=$(dirname $(realpath $0))
 
-pip install -r "${CURRENT_DIR}/customization/requirements.txt"
+# pip install -r "${CURRENT_DIR}/customization/requirements.txt"
 
-"${CURRENT_DIR}"/update_symlinks.sh
+# "${CURRENT_DIR}"/update_symlinks.sh
 
-"${CURRENT_DIR}"/customization/lib_map.sh
+# "${CURRENT_DIR}"/customization/lib_map.sh
```

Comment:
All this commenting-out was left over from local testing, sorry. Will push a commit reverting it.